Sources/WordPressKit/Services/PostServiceRemoteREST+Extended.swift

```diff
@@ -59,6 +59,23 @@ extension PostServiceRemoteREST: PostServiceRemoteExtended {
             }
         }
     }
+
+    public func createAutosave(forPostID postID: Int, parameters: RemotePostCreateParameters) async throws -> RemotePostAutosaveResponse {
```

Comment:
Can be included to `PostServiceRemoteExtended` protocol for consistency

Reply:
It's only available on `PostServiceRemoteREST` and not the XMLRPC version.

Reply:
Got it